### PR TITLE
Update to the topic 4.1

### PR DESCRIPTION
**Pull request by Gowtham-GitHub**
1.Pricing section
2.ENI - details added
3.Status Check
4.Enhanced Networking
5.SSH
6.Security Groups
7.Spot Instances

### CODE REVIEW

File: saa/4.1. Compute - EC2.md
Comment (undergroundwires) [inline saa/4.1. Compute - EC2.md]: whitespace before `(`

File: saa/4.1. Compute - EC2.md
Comment (undergroundwires) [inline saa/4.1. Compute - EC2.md]: Very nice addition. One small thing: aws should be AWS (capitalized)

File: saa/4.1. Compute - EC2.md
Comment (undergroundwires) [inline saa/4.1. Compute - EC2.md]: Eastic -> Elastic

**Review by undergroundwires — CHANGES_REQUESTED** [review]
Awesome job, the added information is clear and gives key knowledge and thanks for fixing so many mistakes. There are some minimal typos.

File: saa/4.1. Compute - EC2.md
Comment (Gowtham-GitHub) [reply saa/4.1. Compute - EC2.md]: 👍

File: saa/4.1. Compute - EC2.md
Comment (Gowtham-GitHub) [reply saa/4.1. Compute - EC2.md]: Thanks.

File: saa/4.1. Compute - EC2.md
Comment (Gowtham-GitHub) [reply saa/4.1. Compute - EC2.md]: Great

**Review by Gowtham-GitHub — COMMENTED** [review]
Overall, Great review by you @undergroundwires

**Comment by undergroundwires** [comment]
Thanks @Gowtham-GitHub ! Happy to hear you can handle some feedback, worked with many people getting mad before haha

I will merge the PRs when you have time to do the corrections. Just in case you did not do that before: You'll need to go to your fork, respective branches that PR's are coming from (e.g. [patch-11](https://github.com/Gowtham-GitHub/AWS-in-bullet-points/tree/patch-11) for this one) and update the content with new commits in each branch/PR, then they'll appear here in PR and I'll merge them. Let me know if you have any questions, and good luck 🙂.

**Comment by undergroundwires** [comment]
Hi @Gowtham-GitHub , it's been a while and I'd love to get these PRs merged as they're just pretty good improvements. If you do not have time I can do the changes for you but right now I cannot edit your PR, would you like to take a look at those to fix them or give me permissions so I can fix them?

You just need to check this one in the PR page:
image

Have a great week and thanks again!

**Comment by Gowtham-GitHub** [comment]
Hi undergroundwires, 
I'm sorry for the delay. 
I'll complete it within this week my friend. Thanks for your appreciation. 
--
Regards,
Gowtham Monday, 21 September 2020, 05:13am +05:30 from undergroundwires  notifications@github.com :

**Comment by Gowtham-GitHub** [comment]
I couldn't find an option to assign it to you undergroundwires. 
But the option you sent me, i.e., Allow edits by maintainers is checked by default for all the four pull requests. 
Please tell me how to merge these PRs to the main branch. 
--
Regards,
Gowtham Monday, 21 September 2020, 05:13am +05:30 from undergroundwires  notifications@github.com :

**Comment by Gowtham-GitHub** [comment]
Please merge the PRs. 
Let me know if I'm required to do any changes. 
--
Regards,
Gowtham Wednesday, 08 July 2020, 01:11am +05:30 from undergroundwires  notifications@github.com :

**Comment by undergroundwires** [comment]
Hi @Gowtham-GitHub , sorry for coming back late. We just need to apply the feedback onto your branch. So one option is to go to your branch called `patch-11` because you're merging from that branch in this PR.
image
So when you go to [your branch in your fork](https://github.com/Gowtham-GitHub/AWS-in-bullet-points/tree/patch-11)  you should be able to update the file [`/4.1. Compute - EC2.md`](https://github.com/Gowtham-GitHub/AWS-in-bullet-points/blob/patch-11/saa/4.1.%20Compute%20-%20EC2.md) in that branch according to my feedback. It'd probably take a short time as it's just changing a few characters, the hardest part is to figure out how we do the updates here 😀 Feel free to take look at it whenever you have time.

Thanks for your collaboration again. Thanks for double checking permissions. I'll see why I cannot do the edits and try it also meanwhile 🤔

**Comment by undergroundwires** [comment]
Hi @Gowtham-GitHub , I mastered Git + GitHub after the PRs 😀 So GitHub has a utility called `gh` that allows to check into PRs and do changes, it was actually pretty easy. Thanks again man, double checking after some time, all updates look very good and logical. Hope you're safe and will be safe during winter and will have a great carrier with AWS!

**Comment by Gowtham-GitHub** [comment]
Happy to hear that. 😊
My contributions will continue. 
Thanks for the appreciate man. 
--
Regards,
Gowtham Monday, 16 November 2020, 02:28pm +05:30 from undergroundwires  notifications@github.com :

**Pull request merged**
